GoogleDataTransport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -171,7 +171,8 @@ gdt_cct_ClientInfo GDTCCTConstructClientInfo() {
   clientInfo.ios_client_info = GDTCCTConstructiOSClientInfo();
```

Comment:
Should tvOS and Catalyst be counted with iOS?

Reply:
Good question. I think yes because, right now, there are only two different protos for Apple platforms to choose from (`IosClientInfo` and `MacClientInfo`). And all fields of `IosClientInfo` can be completed on tvOS and macCatalyst because they similar enough to iOS. 

With this change, GDT data from macOS is distinguished by the presence of the `MacClientInfo` and GDT data from iOS/tvOS/macCatalyst clients could be distinguished by the `IosClientInfo`'s `model` field.